atomic_reactor/schemas/user_params.json

```diff
@@ -41,10 +41,7 @@
     "reactor_config_override": {"type": "object"},
     "release": {"type": "string"},
     "scratch": {"type": "boolean"},
-    "signing_intent": {
-      "type": "array",
-      "items": {"type": "string"}
-    },
+    "signing_intent": {"type": "boolean"},
```

Comment:
Isn't this a string?

Reply:
:param signing_intent: bool, True to sign the resulting image

per the docs for osbs-client/build/buildrequestv2, it's a boolean.

Reply:
Ok, that is also wrong. The value is used by PluginsConfiguration to set the `signing_intent` parameter of the `resolve_composes` plugin of atomic reactor. And that value is then used as a string.

Reply:
Sure, fixed.